src/app/components/shared/formly/image-input.component.ts

```diff
@@ -20,18 +20,27 @@ import { asFormControl } from "./helper";
       >
         <!-- Ensure only one file can be selected in input -->
         <input
+          #imageInput
           type="file"
           accept="image/*"
           class="form-control"
           [formControl]="asFormControl(formControl)"
           [formlyAttributes]="field"
           (ngModelChange)="readFile()"
         />
+
+        <button
+          type="button"
+          (click)="removeImage()"
+          class="btn btn-outline-danger"
+        >Remove</button>
       </div>
     </div>
   `,
 })
 export class ImageInputComponent extends FieldType {
+  @ViewChild("imageInput", {static: false})
+  public inputVar: ElementRef;
```

Comment:
bad name, suggestion:
```suggestion
  public imageInput: ElementRef;
```